assets/src/dashboard/app/views/myStories/content/storiesView/index.js

```diff
@@ -163,7 +163,7 @@ function StoriesView({
         message: sprintf(
           /* translators: %s: story title. */
           __('%s has been copied to your clipboard.', 'web-stories'),
-          story.title.length > 0 ? story.title : __('(no title)', 'web-stories')
+          story.title.length > 0 ? story.title : __('Untitled', 'web-stories')
```

Comment:
This is just to bring this in line with what WordPress does. WordPress displays `Untitled` everywhere nowadays, not `(no title)`

Reply:
```suggestion
          story.title.length > 0 ? titleFormatted( story.title )
```
Save another translation and reuse the code?
Could this just be?

Reply:
`titleFormatted` is a util from dashboard

Reply:
And this line is from the dashboard....

Reply:
lol, whoops.